<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1147</width>
    <height>877</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="portBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="freshButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onEMGButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect EMG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="countNumber">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="emgNumber">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c1Plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="beginButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="squaretestButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SquareTest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>760</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="GestureEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Mav0">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>750</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="Threshold">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>750</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>760</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>mav0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>760</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>threshold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1P">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2P">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3P">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox4P">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4P</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1N">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1N</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2N">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2N</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3N">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3N</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox4N">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>800</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4N</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRLD">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>800</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set RLD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>800</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read Register</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddr">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>800</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>800</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Addr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noiseButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NoiseTest</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normalMeaButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>NormalMeasure</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c2Plot" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c3Plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c4Plot" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c5Plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c6Plot" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>400</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c7Plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="c8Plot" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="elbowPlot" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>120</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="shoulderPlot" native="true">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>330</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="editorButton">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>30</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="onIMUButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect IMU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>30</y>
      <width>80</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibration</string>
    </property>
   </widget>
   <widget class="QTableView" name="moveTable">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>540</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>700</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>700</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>700</y>
      <width>441</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lengthLabel">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>700</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>700</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1147</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
